pkg/cloud/aws/actuators/machine/utils.go

```diff
@@ -204,10 +204,14 @@ func EncodeProviderStatus(codec codec, awsStatus *providerconfigv1.AWSMachinePro
 
 // IsMaster returns true if the machine is part of a cluster's control plane
 func IsMaster(machine *clusterv1.Machine) bool {
+	var isMaster bool
+	if machineRole, exists := machine.ObjectMeta.Labels[providerconfigv1.MachineRoleLabel]; exists && machineRole == "master" {
+		isMaster = true
+	}
 	if machineType, exists := machine.ObjectMeta.Labels[providerconfigv1.MachineTypeLabel]; exists && machineType == "master" {
```

Comment:
Oh yeah, this is a leftover. We don't use machine type, we use node role instead. So this can be dropped. @enxebre does the installer sets machine type at all? If so, what is its meaning?

Reply:
They set both at the moment, but we can drop it from the installer then.
https://github.com/openshift/installer/blob/30bb25ac57d7c7d3dae519186cbfca9af8aeaca2/pkg/asset/machines/aws/machines.go#L55